feature-libs/misc/storefinder/core/public_api.ts

```diff
@@ -7,3 +7,4 @@ export * from './service/index';
 export * from './store/actions/index';
 export * from './store/selectors/index';
 export * from './store/store-finder-state';
+export * from './store/store-finder-store.module';
```

Comment:
@dunqan was this removed in the final merge of the storefinder ticket?

Reply:
it was removed